### ISSUE TITLE
upgrade gradle

### ISSUE THREAD

**Issue by psibre** [marytts/gradle-marytts-voicebuilding-plugin#52]
Currently, upgrading the Gradle wrapper to a version newer than 2.9 is blocked, as it triggers errors such as:
```
Gradle suite > Gradle test > de.dfki.mary.voicebuilding.BuildLogicFunctionalTest.testConfigurations FAILED
    java.lang.NoSuchMethodError: org.gradle.tooling.internal.consumer.DefaultBuildLauncher.withInjectedClassPath(Lorg/gradle/testkit/jarjar/org/gradle/internal/classpath/ClassPath;)Lorg/gradle/tooling/internal/consumer/AbstractLongRunningOperation;
        at org.gradle.testkit.runner.internal.ToolingApiGradleExecutor.run(ToolingApiGradleExecutor.java:98)
        at org.gradle.testkit.runner.internal.DefaultGradleRunner.run(DefaultGradleRunner.java:270)
        at org.gradle.testkit.runner.internal.DefaultGradleRunner.build(DefaultGradleRunner.java:219)
        at org.gradle.testkit.runner.internal.DefaultGradleRunner$build$2.call(Unknown Source)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:117)
        at de.dfki.mary.voicebuilding.BuildLogicFunctionalTest.testConfigurations(BuildLogicFunctionalTest.groovy:161)
```
This should be fixed!

**Comment by psibre** [comment]
For reference, this is tracked upstream as [GRADLE-3433](https://issues.gradle.org/browse/GRADLE-3433).

**Comment by psibre** [comment]
This issue was fixed upstream, released as part of Gradle 2.14.

**Comment by psibre** [comment]
This should have been closed along with #64
